modules/openapi-generator/src/test/java/org/openapitools/codegen/python/PythonClientExperimentalTest.java

```diff
@@ -0,0 +1,297 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
```

Comment:
@spacether I don't think we need line 3 as per https://opensource.stackexchange.com/a/7301/12997

Reply:
Removed the SmartBear copyright line